frontend/packages/metal3-plugin/src/components/host-detail.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import { Row, Col } from 'patternfly-react';
+
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
+import { DetailsPage } from '@console/internal/components/factory';
+import { navFactory, SectionHeading, Timestamp } from '@console/internal/components/utils';
+import { units } from '@console/internal/components/utils/units';
+import { getName } from '@console/shared';
+
+import { BaremetalHostModel } from '../models';
+import MachineCell from './machine-cell';
+import {
+  getHostNICs,
+  isHostOnline,
+  getHostDescription,
+  getHostBMCAddress,
+  getHostCPU,
+  getHostRAM,
+  getHostTotalStorageCapacity,
+} from '../selectors';
+
+type BaremetalHostDetailPageProps = {
+  namespace: string;
+  name: string;
+  match: any;
+};
+
+const BaremetalHostDetails: React.FC<{ obj: K8sResourceKind }> = ({ obj }) => {
+  const { creationTimestamp } = obj.metadata;
+  const nics = getHostNICs(obj);
+  const online = isHostOnline(obj);
+  const ips = nics.map((nic) => nic.ip).join(', ');
+
+  const statusIconClasses = classNames('fa fa-refresh', { 'co-icon-and-text__icon': online });
+
+  const totalCapacity = units.humanize(
+    // The value from the selector is in GB
+    getHostTotalStorageCapacity(obj) * 1024 ** 3,
+    'decimalBytes',
+    true,
+  ).string;
+
+  return (
+    <div className="co-m-pane__body">
+      <SectionHeading text="Baremetal Host Overview" />
+      <Row>
+        <Col lg={6} md={6} sm={6} xs={6} id="name-description-column">
```

Comment:
```
<Col sm={6} xs={12}>
``` 
Should be sufficient to get 2 columns on all sizes except `xs` which will stack items into a single column

Reply:
Done